homeassistant/components/netatmo/sensor.py

```diff
@@ -267,7 +265,7 @@ class NetatmoSensorEntityDescription(SensorEntityDescription, NetatmoRequiredKey
         name="Power",
         netatmo_name="power",
         entity_registry_enabled_default=True,
-        native_unit_of_measurement=POWER_WATT,
+        native_unit_of_measurement=UnitOfPower.WATT,
```

Comment:
I've kept it as it was, but other weather integrations use `IRRADIATION_WATTS_PER_SQUARE_METER`

Reply:
Without response from the code owner, I think it's safer to assume to the current unit is correct.